tests/test_efa_srd.py

```diff
@@ -12,6 +13,29 @@
 import tests.utils as u
 
 
+class CqEventsSRD(SRDResources):
+    def __init__(self, dev_name, ib_port, gid_index):
+        super().__init__(dev_name, ib_port, gid_index, e.IBV_QP_EX_WITH_SEND)
+
+    def create_cq(self):
+        self.comp_channel = CompChannel(self.ctx)
+        self.cq = CQ(self.ctx, self.num_msgs, None, self.comp_channel)
+        self.cq.req_notify()
+
+
+class CqEventsSRDTestCase(EfaRDMATestCase):
+    def setUp(self):
+        super().setUp()
+        self.iters = 100
+        self.server = None
+        self.client = None
```

Comment:
BTW, no need to initiate server and client as it's already done in RDMATestCase.__init__()
Except for that, it looks good to me.

Reply:
OK, removed.